src/components/ec-table/ec-table.vue

```diff
@@ -137,81 +137,81 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/index';
+<style>
+@import '../../styles/tools/scrollbars';
+
+.common-cell-layout {
+  &:last-of-type {
+    padding-right: 16px;
+  }
+}
 
 .ec-table-scroll-container {
-  overflow: auto;
+  @apply tw-overflow-auto;
 
-  @include small-scrollbar;
+  @mixin small-scrollbar;
 }
 
 .ec-table {
+  @apply tw-border-none;
+  @apply tw-pt-16;
+  @apply tw-w-full;
+  @apply tw-relative;
+
   border-spacing: 0;
-  border: none;
-  padding-top: 16px;
-  width: 100%;
-  position: relative; // We need to reset the z-index so the sticky columns and header will not compete with the outer world
-  z-index: 0;
-
-  %common-cell-layout {
```

Comment:
But why do we remove this?

Reply:
I moved it to the top of the style block. The compiler was complaining that I could not define it here